src/driver/test_point_aggregator.py

```diff
@@ -140,21 +140,22 @@ def aggregate(
     Returns:
       A single row DataFrame representing the values of the statistics listed in keys.
     """
-    stats = {"model_succeeded": [1], "model_exception": [""]}
+    stats = {"model_succeeded": [1], "model_exception": ["None"]}
```

Comment:
Why this change?

Reply:
```python
result.to_csv(na_rep="NaN")
```
This will make not only `np.NaN` but also empty cells to `"NaN"`. To avoid the wrong conversion for the empty cells in `model exception`, I put "None" into the cells instead.
I think this is reasonable because an empty cell in a `dataframe` usually means missing data. Every cell should have a clear value.

Reply:
sgtm